packages/eas-cli/src/project/isEasEnabledForProject.ts

```diff
@@ -0,0 +1,21 @@
+import { apiClient } from '../api';
+
+/**
+ * Checks if the project is allowed for using EAS services.
+ * THIS IS A TEMPORARY STEP NEEDED FOR OPEN PREVIEW
+ * @returns boolean
+ */
+export async function isEasEnabledForProjectAsync(projectId: string): Promise<boolean> {
+  try {
+    const {
+      data: { enabled },
+    } = await apiClient.get(`projects/${projectId}/eas-enabled`).json();
+    return enabled;
+  } catch (err) {
+    if (err.response.statusCode === 404) {
+      return true;
+    } else {
+      throw err;
+    }
```

Comment:
```suggestion
  } catch (error) {
    if (error.response?.statusCode === 404) {
      return true;
    } else {
      throw error;
    }
```
It's possible for the `try` block to throw an error that doesn't originate from the HTTP library, so the `?` in `err.response?.statusCode` is needed. Additionally used a full word for the variable name.